fabfile/run.py

```diff
@@ -17,3 +25,135 @@ def regression():
     'Runs Citus\' regression tests'
     with cd(config.paths['citus-repo']):
         run('make check')
+
+@task
+@runs_once
+@roles('master')
+def dml_tests(*args):
+    'Runs pg-bench dml tests using the given config options'
+    dir_path = '/home/ec2-user/'
```

Comment:
I don't like hard coded paths. Is there a way you can get this from the environment ?

Reply:
```python
from os.path import expanduser
home = expanduser("~")
```

gives you home directory of running user